velox/vector/tests/VectorTest.cpp

```diff
@@ -3517,5 +3517,17 @@ TEST_F(VectorTest, flatAllNulls) {
   }
 }
 
+TEST_F(VectorTest, setType) {
+  auto type = ROW({"aa"}, {BIGINT()});
+  auto vector = BaseVector::create(type, 1'000, pool());
+  auto newType = ROW({"bb"}, {BIGINT()});
+  vector->setType(newType);
+  EXPECT_EQ(vector->type()->toString(), newType->toString());
+  auto failedType = ROW({"bb"}, {VARCHAR()});
```

Comment:
nit: add empty line to visually separate logical test cases

Reply:
@mbasmanova Added.